### DISCUSSION TITLE
Is it possible to load a Livewire component like `@livewire(\App\Http\Livewire\Foo::class)`?

### DISCUSSION THREAD

**Discussion by ralphjsmit** [livewire/livewire#4196] [1. Help] [answered]
I'd like to load a Livewire component like this: 
```
@livewire(\App\Http\Livewire\Foo::class)
```

Is that possible? I don't see it in the docs, that's why I ask, also because I know that it _is_ possible in places like `->assertSeeLivewire()`.

**Comment by joshhanley** [comment, marked as answer]
@ralphjsmit yep, that should work!

May I ask, why do you want to do that?

**Reply by ralphjsmit** [reply]
Hey Josh, thanks!

I'm trying to wire some complex Livewire setup together that supports modals and slideovers in combination with the new Filament Forms.

Consider the following scenario:

```php
// Parent component
public function mount($childComponentClass) { $this->childComponentClass = $childComponentClass; }

public function render() 
{
    return <<<'blade'
           <div>
               @livewire($this->childComponentClass)
               <button type="submit">{{ $this->childComponentClass::BUTTON_NAME }}</button>
           </div>
       blade;
```
```php
// Child component

CONST BUTTON_NAME = 'Confirm';
```

**Reply by joshhanley** [reply]
@ralphjsmit ah yep, that makes sense! Thanks for sharing, always curious to how others are using things 🙂
